File: eighth/c/5.c
#include <stdio.h>
#include <string.h>

void gerarEmail(const char *nomeCompleto, char *email) {
    char primeiroNome[50];
    char sobrenome[100];
    
    sscanf(nomeCompleto, "%s", primeiroNome);
    int i = strlen(primeiroNome) + 1;
    strncpy(sobrenome, nomeCompleto + i, 100);
    for (i = 0; sobrenome[i]; i++) {
        sobrenome[i] = tolower(sobrenome[i]);
    }
    
    sprintf(email, "%c%s@lp.com.br", tolower(primeiroNome[0]), sobrenome);
}

int main() {
    char nomeCompleto[150] = "Sr. Fulano de Tal";
    char email[150];
    gerarEmail(nomeCompleto, email);
    printf("%s, seu email é: %s\n", nomeCompleto, email);
    
    return 0;
}

File: eighth/python/5.py
def gerar_email(nome_completo):
    partes_nome = nome_completo.split()
    primeiro_nome = partes_nome[0]
    sobrenome = " ".join(partes_nome[1:])
    email = f"{primeiro_nome[0].lower()}{sobrenome.replace(' ', '').lower()}@lp.com.br"
    return email

# Exemplo de uso
nome_completo = "Sr. Fulano de Tal"
email = gerar_email(nome_completo)
print(f"{nome_completo}, seu email é: {email}")
